src/insert/spacefinder/article.ts

```diff
@@ -350,8 +381,8 @@ const addMobileInlineAds = (fillSlot: FillAdSlot): Promise<boolean> => {
 
 	const insertAds: SpacefinderWriter = async (paras) => {
 		const slots = paras.map(async (para, i) => {
-			const name = i === 0 ? 'top-above-nav' : `inline${i}`;
-			const type = i === 0 ? 'top-above-nav' : 'inline';
+			const name = `inline${i}`;
```

Comment:
```suggestion
			const name = `inline${i + 1}`;
```

Reply:
Yep you're totally right 🤦‍♀️ Great spot!